cloud/user/user.go

```diff
@@ -0,0 +1,49 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/software/workspace"
```

Comment:
we should avoid referring to `software/workspace` in `cloud` packages, as both of these packages are maintained by different teams

Reply:
Good catch and will fix.